test/e2e/e2e_test.go

```diff
@@ -14,8 +14,8 @@ import (
 var _ = Describe("policy validation", func() {
 	Describe("When enforcing operator policy", func() {
 		var (
-			// TODO: replace this with a project-accessible namespace once identified.
-			goodImage = "quay.io/komish/preflight-test-bundle-passes:latest"
+			// TODO: replace the failure case with crafted examples.
+			goodImage = "quay.io/opdev/simple-demo-operator-bundle:v0.0.2"
```

Comment:
Out of curiosity: do we plan to have a pipeline or a manual procedure to create/update this bundle image?